### PR TITLE
Support `resource_class` parameter for UDFs

### PR DESCRIPTION
**Pull request by johnkerl**
As requested by @leipzig .

The goal is to be able to request a UDF to be run on an UDF runner with `resource_class="large"`, being an instance with 8GB of RAM rather than the default 2GB of RAM.

Following the recipe I wrote a year ago at https://tiledb-inc.github.io/TileDB-Cloud-R/articles/HowToUpdateAPI.html.

* `R/generic_udf.R`, `R/multi_array_udf.R`, and `R/sql_parameters.R` were written by the OpenAPI code generator as described at the link just above.
* `R/manual_layer*.R` I hand-edited to feed forward the new `resource_class` parameter.
* I ran `roxygen2::roxygenise()` to get updated roxygen artifacts for the three `R/manual_layer_*.R` files modified on this PR.

### CODE REVIEW

**Comment by eddelbuettel** [comment]
Bit gnarly too read: who or what is actually dealing with the resource?  It looks as if this is a parameter being passed on the Cloud API?

**Comment by johnkerl** [comment]
@eddelbuettel yes this is a parameter being passed to the cloud API to select a UDF-runner instance having 8GB RAM rather than 2GB RAM.

As noted in the description field of this PR, the `.R` files not starting with `manual_layer_` are autogenerated by the OpenAPI code generator. Hence all the backticks I did not keystroke out by hand.

The gnarl is not my own.

**Comment by eddelbuettel** [comment]
Yep, I know. The PR was just lacking even a single sentence stating that, and I am less in the cloud env than you are, hence the innocent question -- and thanks for clarifying. 

It would be easier to approve if we 'got the green back'.  I think when I was making changes a while back I had to update the access, we now seem to fail on 

> Exited at #18: No TILEDB_REST_UNIT_TEST_NAMESPACE_TO_CHARGE in environment]

Any chance to address that?

**Comment by johnkerl** [comment]
> Any chance to address that?

I will try to figure it out. Again, it's been a year since I touched any of this.

File: R/manual_layer_udf.R [hunk 1]
```diff
@@ -147,14 +152,16 @@ execute_generic_udf <- function(udf=NULL, registered_udf_name=NULL, args=NULL, r
 ##' logged-in user's username will be used for the namespace.
 ##'
 ##' @param language If omitted, defaults to \code{"r"}. Can be set to \code{"python"}
-##" when executing registered Python UDFs.
+##' when executing registered Python UDFs.
+##'
+##' @param resource_class TO DO TYPE ME UP
```
Comment (aaronwolen) [inline R/manual_layer_udf.R:157]:
```suggestion
##' @inheritParams execute_generic_udf
```

File: R/manual_layer_udf.R [hunk 2]
```diff
@@ -283,13 +293,15 @@ execute_array_udf <- function(array, udf=NULL, registered_udf_name=NULL, selecte
...
```
Comment (aaronwolen) [inline R/manual_layer_udf.R:298]:
```suggestion
##' @inheritParams execute_generic_udf
```

**Review by leipzig — APPROVED** [review]

**Pull request merged**